.github/workflows/ci.yml

```diff
@@ -106,7 +106,7 @@ jobs:
         with:
           activate-environment: modin
           environment-file: environment.yml
-          python-version: 3.6
+          python-version: 3.8
```

Comment:
Please change tests in a different PR

Reply:
Because now pandas 1.2.0 supports Python 3.7.1 and higher there was dependencies conflict in `test-headers` job. I changed python version for testing purposes to see that helps. Okay, I will change that in a different PR.

Reply:
I see, yes it would be good to change the tests separately to have more fine-grained commit docs.